Makefile.am

```diff
@@ -50,3 +50,10 @@ dist_noinst_DATA = \
 	AUTHORS COPYING README \
 	example.c \
 	testinput.c
+
+if CAN_COMPILE_EXAMPLE
+EXTRA_PROGRAMS = example
+example_SOURCES = example.c
+example_LDADD = $(LDADD) $(LIBGCRYPT_LIBS) $(lib_LTLIBRARIES)
```

Comment:
It works for me (slackware-current). I haven't tested it anywhere else. Did you mean something different in your earlier comment where you said "the LDADD should be combined with the current one"?